app/pb_kits/playbook/pb_range_slider/_range_slider.scss

```diff
@@ -0,0 +1,102 @@
+[class^=pb_range_slider] {
+
+  @mixin track {
+    height: 4px;
+    -webkit-appearance: none;
+    appearance: none;
+    margin-top: -1px;
+    background-color: $primary_action;
+    border-radius: 2px;
+  }
+
+  @mixin thumb {
+    width: 16px;
+    -webkit-appearance: none;
+    appearance: none;
+    height: 16px;
+    cursor: pointer;
+    background: #fff;
+    box-shadow: make-progress($border_light);
+    border: 3px solid $primary_action;
+    border-radius: 50%;
+    position: relative;
+    top: -6px;
+    transition: box-shadow 0.2s ease-in-out;
+    &:after {
+      content: '';
+      background: #808080;
+      display: block;
+      width: 100vw;
+      height: 4px;
+    }
+  }
+
+  @function make-progress($color) {
```

Comment:
This is the reason I think we should do this in JS. Looking at the output of the background it adds a significant amount of CSS to the page.

Reply:
FYI, I'll get back to this at some point today.